### ISSUE TITLE
Division by zero

### ISSUE THREAD

**Issue by William-Lake** [William-Lake/StockNewsAnalyzer#14]
Sometimes when a stock is being rated the following error gets thrown:

```
2018-11-30 06:11:34,170 news_rater.py:score_articles:112          ERROR    division by zero
```

I'm pretty sure it's from the news_rater.__score_content method:

```python
sentences = tokenize.sent_tokenize(content_text)

total_compound_score = 0

for sentence in sentences: total_compound_score += self.analyzer.polarity_scores(sentence)['compound']

raw_score = total_compound_score / len(sentences)

return raw_score
```

If there are no sentences, then trying to calculate the raw_score will throw this error.

The larger (and more important) issue is that this isn't caught in _this_ method, it's caught the method listed in the error: score_articles. That catch makes score_articles skip the current article. **BUT** that means the `time.sleep()` piece never gets ran. This means if there are repeated errors like this (or any for that matter,) you'll end up spamming the site and potentially get your ip banned from whatever site you're trying to gather data from.

**Comment by William-Lake** [comment]
Resolved in most recent commit.
